desktopexporter/internal/server/server.go

```diff
@@ -0,0 +1,156 @@
+package server
```

Comment:
Just as a note, when refactoring the project and moving things around using `git mv` instead of `mv` will retain history and clean up the PR diff view. I can't easily tell if anything changed between this file and the `server.go` that was deleted.

This is also a good way of getting bugs assigned to you in a big codebase, because even if you didn't write most of the code, now all of the lines `git blame` to you 😅